change/@uifabric-styling-2019-07-31-14-05-43-phkuo-leftNavRedlines.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
is adding new slots a `patch`? Technically it’s adding new API so maybe `minor`?

Reply:
good point